bridge/src/main/java/com/bridge/game/Game.java

```diff
@@ -1,5 +1,35 @@
 package com.bridge.game;
 
-public abstract class Game {
-    public abstract boolean isGameWon();
-}
+import com.bridge.processinputhandler.InputVerifier;
+
+/**
+ * Main game loop class.
+ */
+public class Game {
```

Comment:
The old abstract class  `Game` is meant to be renamed as `GameSettings`.
  Please create another `Game` class